site/content/entry/citra-progress-report-2018-Q1.md

```diff
@@ -0,0 +1,233 @@
+    "renderer-settings-new.png=New Graphics Tab" >}}
+
+During it's time in Canary, a lot of testing was done by our contributors. They tested many number of
+games, under varied hardware conditions and submitted valuable telemetry data, crash logs, etc.,
+which helped us in fixing issues with the hardware shader support.
+
+**Note**:
+Unfortunately, due to driver issues with AMD GPUs and inadequate OpenGL support on macOS, the hardware
+shader might not perform better than the CPU renderer under these conditions. However, worry not, as
+research is being done into possible fixes that might come around sooner than you think.
+
+Read more about hardware renderer improvements in detail [here](https://citra-emu.org/entry/improvements-to-hardware-renderer/).
+
+## Continuous Integration (CI) Improvements ([here](https://github.com/citra-emu/citra/pull/3647), [there](https://github.com/citra-emu/citra/pull/3649), [this](https://github.com/citra-emu/citra/pull/3706), and [that](https://github.com/citra-emu/citra/pull/3744)) by [liushuyu](https://github.com/liushuyu)
+
+One of our major time and resource consuming parts of the project were our build generating environments.
+We use Travis and Appveyor to release fully functional builds for download. However, the time it took
+for the builds to be created was too long and if it was stopped abruptly, it took even more time to
+create the builds again. Thanks to [liushuyu](https://github.com/liushuyu)'s contributions, we were
+able to optimize this whole process and make it very efficient to build and release latest changes to
+Citra.
+
+## Logging System Rewrite ([here](https://github.com/citra-emu/citra/pull/3449), [there](https://github.com/citra-emu/citra/pull/3533) and [that](https://github.com/citra-emu/citra/pull/3568)) by [jroweboy](https://github.com/jroweboy) and [daniellimws](https://github.com/daniellimws)
+
+Our old logging system has been becoming obsolete. So, a newer and efficient logging system based on
+fmtlib was designed and implemented by [jroweboy](https://github.com/jroweboy) and [daniellimws](https://github.com/daniellimws).
+This gets rid of the ugly console window and made many under the hood improvements to the logging system.
+This will make identifying and fixing issues a whole lot faster. After the implemetation was done,
+migrating our codebase to the new system was a huge endeavour shared by [daniellimws](https://github.com/daniellimws),
+and many others.
+
+## [citra-qt: Camera Support](https://github.com/citra-emu/citra/pull/3566) by [zhaowenlan1779](https://github.com/zhaowenlan1779)
+
+One of the highly requested features of Citra has been the support for using the system camera to
+scan QR codes, etc. [Zhaowenlan1779](https://github.com/zhaowenlan1779) took it upon himself to
+provide Citra with this feature. By making use of the available Qt libraries, he worked on the
+implementation for weeks. With some collaboration from other contributors, he was finally able to
```

Comment:
This "With some collaboration from other contributors" sentence kinda feels like padding.